web_notify/models/res_users.py

```diff
@@ -1,46 +1,92 @@
+# pylint: disable=missing-docstring
 # Copyright 2016 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, exceptions, fields, models, _
+from enum import Enum
+
+from odoo import _, api, exceptions, fields, models
+
+DEFAULT_MESSAGE = "Default message"
+
+
+class NotificationType(Enum):
```

Comment:
I don't think you need this, you can just define N costants or simply use namedtuple since there's actually no enumeration here